api/cds.proto

```diff
@@ -14,6 +14,8 @@ import "google/protobuf/duration.proto";
 import "google/protobuf/struct.proto";
 import "google/protobuf/wrappers.proto";
 
+import "validate/validate.proto";
```

Comment:
Please make sure all items have some text, even if basic. There is some stuff missing: https://554-90675549-gh.circle-artifacts.com/0/source/generated/docs/api-v2/cds.proto.html. In general I also prefer line breaks between items and comments as it's easier to read for people editing the file.